src/utils/embedded/utils/routes/embedded-routes.utils.ts

```diff
@@ -9,7 +11,11 @@ const routeTypeByLocationPrefix: Record<string, RouteType> = {
   "auth-request": "auth-request",
 };
 
-export function locationToRouteType(path: WanderRoutePath): RouteType {
+export function locationToRouteType(path: WanderRoutePath, authStatus: AuthStatus): RouteType {
+  if (path === EmbeddedPaths.SupportUnpartitionedStateMissing) {
+    return authStatus === "unlocked" ? "default" : "auth";
```

Comment:
This view can be shown before auth and after auth, so we change the layout to match the rest of the views.